### PR TITLE
Add vCluster Iron Bank

### PR DESCRIPTION
**Pull request by mridul549**
### Description of the change

Add vCluster Iron Bank Image

### Applicable issues

  - fixes #

### Checklist

- [X] Added to the `README.md` using [readme-generator](https://github.com/rapidfort/community-images/readme-generator)

### CODE REVIEW

File: community_images/vcluster/ironbank/k8s_before.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+docker tag "${REPO_PATH}":"${TAG}" "${REPO_PATH}":latest
+
+# load the image in minikube cluster
+minikube image load "${REPO_PATH}":latest
+
+# load the image in kind cluster
+# kind load docker-image --name rapidfort-platform "${REPO_PATH}":latest
+
+# Installing the vcluster CLI to communicate with the created vcluster
+curl -L -o vcluster "https://github.com/loft-sh/vcluster/releases/latest/download/vcluster-linux-amd64" && sudo install -c -m 0755 vcluster /usr/local/bin
```
Comment (AnmolVirdi) [inline community_images/vcluster/ironbank/k8s_before.sh:25]: Can we run this without sudo?

Comment (AnmolVirdi) [reply community_images/vcluster/ironbank/k8s_before.sh:25]: Also, let's note this down in the related ticket.

**Review by AnmolVirdi — CHANGES_REQUESTED** [review]

File: community_images/vcluster/ironbank/k8s_before.sh (hunk 1 above)
Comment (mridul549) [reply community_images/vcluster/ironbank/k8s_before.sh:25]: > Can we run this without sudo?

Yes, works without sudo as well.

Comment (mridul549) [reply community_images/vcluster/ironbank/k8s_before.sh:25]: > Also, let's note this down in the related ticket.

This is also done.

**Review by AnmolVirdi — APPROVED** [review]

**Pull request merged**